### DISCUSSION TITLE
Client libraries for iOS and Android?

### DISCUSSION THREAD

**Discussion by Abdullah-l** [anycable/anycable#163] [Q&A] [answered]
Hello,

What client libraries for iOS(Swift) and Android(Java) can be used with AnyCable?

The libraries I found online for ActionCable are not maintained:
https://github.com/danielrhodes/Swift-ActionCableClient
https://github.com/hosopy/actioncable-client-java

So I'm just wondering how it can be used with those platforms, is it through GraphQL?

**Comment by palkan** [comment, marked as answer]
We use the one mentioned by you for iOS (~1 year ago last time, worked fine). Can't really help with Android. [This one](https://github.com/vinted/actioncable-client-kotlin) (Kotlin) seems not so old.

We're working on the official AnyCable iOS SDK, but it's not there yet.
